app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,49 @@
+<h2 class="text-center font-semibold text-3xl py-5">Sign up</h2>
+
+<div class="form-section flex justify-center my-10">
+  <div class="form-container border border-2 p-10">
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
+
+      <div class="field my-5">
+        <%= f.label :name %><br />
+        <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: 'Username', class: 'border my-3 py-1 px-3' %>
+      </div>
+
+      <div class="field my-5">
+        <%= f.label :email %><br />
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: 'Email', class: 'border my-3 py-1 px-3' %>
+      </div>
+
+      <div class="field my-5">
+        <%= f.label :bio %><br />
+        <%= f.text_area :bio, autofocus: true, autocomplete: "bio", placeholder: 'Bio', class: 'border my-3 py-1 px-3' %>
+      </div>
+
+      <div class="field my-5">
+        <%= f.label :photo %><br />
+        <%= f.text_field :photo, autofocus: true, autocomplete: "photo", placeholder: 'Photo URL', class: 'border my-3 py-1 px-3' %>
+      </div>
+
+      <div class="field my-5">
+        <%= f.label :password %>
+        <% if @minimum_password_length %>
+          <em>(<%= @minimum_password_length %> characters minimum)</em>
+        <% end %><br />
+        <%= f.password_field :password, autocomplete: "new-password", placeholder: 'Password', class: 'border my-3 py-1 px-3' %>
+      </div>
+
+      <div class="field my-5">
+        <%= f.label :password_confirmation %><br />
+        <%= f.password_field :password_confirmation, autocomplete: "new-password", class: 'border my-3 px-3 py-1' %>
+      </div>
+
+      <div class="actions my-5">
+        <%= f.submit "Sign up", class: 'border bg-black text-white px-3 py-1 rounded-md' %>
+      </div>
+
+      <%= render "devise/shared/links" %>
+
+    <% end %>
+  </div>
+</div>
```

Comment:
- [ ] Great job on your app really. Kindly style the login and signup buttons to span the width of the input fields as is in the given wireframe

Given frame
![image](https://github.com/Wils-FOURR-speed/Blog-App-Project/assets/81932478/642ffcda-75bf-4946-91db-d4382356756a)

Your design
![image](https://github.com/Wils-FOURR-speed/Blog-App-Project/assets/81932478/c9dad5c2-12be-4ef5-904d-0ec8df33e6ba)